Functions-request-config.js

```diff
@@ -6,6 +6,7 @@ require("@chainlink/env-enc").config()
 const Location = {
   Inline: 0,
   Remote: 1,
```

Comment:
I'm starting to think that "Remote" is not a very descriptive/user friendly term -- intuitively the DON is remote too.

Should we consider "URL" instead?  Or webHosted to contra-distinguish with DONHosted?